integration_test/open_group_test.js

```diff
@@ -23,25 +23,25 @@ describe('Open groups', function() {
     await common.killallElectron();
   });
 
-  it('works with valid group url', async () => {
+  // reduce code duplication to get the initial join
+  async function joinOpenGroup(url, name) {
     await app.client.element(ConversationPage.globeButtonSection).click();
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
 
-    await app.client
-      .element(ConversationPage.openGroupInputUrl)
-      .setValue(common.VALID_GROUP_URL);
+    await common.setValueWrapper(app, ConversationPage.openGroupInputUrl, url);
     await app.client
       .element(ConversationPage.openGroupInputUrl)
       .getValue()
-      .should.eventually.equal(common.VALID_GROUP_URL);
+      .should.eventually.equal(url);
     await app.client.element(ConversationPage.joinOpenGroupButton).click();
 
     // validate session loader is shown
     await app.client.isExisting(ConversationPage.sessionLoader).should
       .eventually.be.true;
+    // account for slow home internet connection delays...
     await app.client.waitForExist(
       ConversationPage.sessionToastJoinOpenGroupSuccess,
-      9000
+      60 * 1000
```

Comment:
the maximum test timeout is currently 30seconds. you want to increase that value too, isn't it? https://github.com/loki-project/session-desktop/pull/1044/files?file-filters%5B%5D=.js#diff-dd50b247dac58a9b47120e5c4f1b2c46R9

Reply:
we can reduce these down, I didn't need that high but I could see other running the integration tests (maybe outside devs) may need it. 

I'm fine either way tbh.